README.md

```diff
@@ -1,17 +1,33 @@
+A simple Go based Yum metadata parser with fetching capabilities.
+
+Currently this project only parses package information, more may be added in the future. 
+
+
 To use this package simply: 
```

Comment:
```suggestion
To use this package: 
```
You should not use words that express an opinion about how hard or easy something is. Its all relative and add no value. Use of the word "simply" is in any case marketing speak.